cryptoki/tests/basic.rs

```diff
@@ -2378,7 +2484,7 @@ fn kbkdf_feedback_mode() -> TestResult {
     let dkm_length_format = KbkdfDkmLengthFormat::new(
         KbkdfDkmLengthMethod::SumOfKeys,
```

Comment:
Could you explain why `SumOfKeys` is OK here but in the fragment above it was changed to `SumOfSegments`? :thinking: Thanks!